### ISSUE TITLE
Homework feedback - JS1 - week 1

### ISSUE THREAD

**Issue by Swapna-Sahu** [Mekdes1/hyf-homework#4]
Hi, Good Job! 👍 Everything looks prefect.

I have one comments in your code.

Task - Housey-Pricey

" Your friend Peter is considering a house that is 8m wide, 10m deep and 10m high. The garden size is 100m2. The house costs 2.500.000. Your friend Julia is considering a house that is 5m wide, 11m deep and 8m high. The garden size is 70m2. This house costs 1.000.000. "

Here House cost is actual house cost price not buying cost for Peter and Julia. 
so whatever you are calculating is the buying cost for Peter and Julia and you need to compare with actual house cost. 

In short  : 
if (housePrice < 2500000) {
    console.log ( "Peter are paying too much")
}
else {
    console.log ( "The price is reasonable")
}
 Other than this all good.Keep it up 🥇

**Comment by Mekdes1** [comment]
Resolved issue
